.idea/workspace.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should not commit user specific configuration file (i.e. Intellij IDEA files).
Other users may want to open the project in other IDE.
Please get familiar with .gitignore file. This website may be helpful to generate the file: http://gitignore.io